### PR TITLE
Add last-updated to state info tooltip 

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

I think for new users it currently might be a bit confusing/weird what the relative time below the name means, especially given that the tooltip is linked to the "last changed" field, but then only displays the "last updated" time currently.

My idea would be show both last-changed and last-updated to clear up that confusion:

image

image

Without that clarification, a user might think that something is bugged ("Why are two conflicting information shown?").

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/entity/state-info.js [hunk 1]
```diff
@@ -81,11 +81,30 @@ class StateInfo extends LocalizeMixin(PolymerElement) {
               datetime="[[stateObj.last_changed]]"
             ></ha-relative-time>
             <paper-tooltip animation-delay="0" for="last_changed">
-              [[localize('ui.dialogs.more_info_control.last_updated')]]:
-              <ha-relative-time
-                hass="[[hass]]"
-                datetime="[[stateObj.last_updated]]"
-              ></ha-relative-time>
+              <table>
```
Comment (bramkragten) [inline src/components/entity/state-info.js:84]: Don't use a `table`

File: src/components/entity/state-info.js [hunk 2]
```diff
@@ -81,11 +102,30 @@ class StateInfo extends LocalizeMixin(PolymerElement) {
...
+                  <div class="column-name">
+                    [[localize('ui.dialogs.more_info_control.last_changed')]]:
+                  </div>
+                  <div class="column-value">
+                    <ha-relative-time
+                      hass="[[hass]]"
+                      datetime="[[stateObj.last_changed]]"
+                    ></ha-relative-time>
+                  </div>
+                </div>
```
Comment (bramkragten) [inline src/components/entity/state-info.js:116]:
```suggestion
                <div class="row">
                    <span>
                      [[localize('ui.dialogs.more_info_control.last_changed')]]:
                    </span>
                    <ha-relative-time
                      hass="[[hass]]"
                      datetime="[[stateObj.last_changed]]"
                    ></ha-relative-time>
                </div>
```

File: src/components/entity/state-info.js [hunk 3]
```diff
@@ -59,6 +59,27 @@ class StateInfo extends LocalizeMixin(PolymerElement) {
...
+          flex-basis: 100%;
+          flex: 1;
+          padding: 2px;
+        }
+        .column-name {
+          align-items: flex-start;
+        }
+        .column-value {
+          align-items: flex-end;
+        }
```
Comment (bramkragten) [inline src/components/entity/state-info.js:82]:
```suggestion
        .row {
          display: flex;
          flex-direction: row;
          flex-wrap: no-wrap;
          width: 100%;
          justify-content: space-between;
        }
```

**Comment by spacegaier** [comment]
@bramkragten I took over your proposal but added back in some margins since otherwise it is all squished together (both the label and the values as well as the two rows).

Is the one inline `style` definition okay?

Without margins:
image

With margins:
image

File: src/components/entity/state-info.js [hunk 4]
```diff
@@ -81,11 +90,26 @@ class StateInfo extends LocalizeMixin(PolymerElement) {
               datetime="[[stateObj.last_changed]]"
             ></ha-relative-time>
             <paper-tooltip animation-delay="0" for="last_changed">
-              [[localize('ui.dialogs.more_info_control.last_updated')]]:
-              <ha-relative-time
-                hass="[[hass]]"
-                datetime="[[stateObj.last_updated]]"
-              ></ha-relative-time>
+              <div>
+                <div class="row" style="margin-bottom: 2px">
```
Comment (bramkragten) [inline src/components/entity/state-info.js:94]: Add the margin-bottom to the `.row` class and remove it on the `:last-child`

**Comment by spacegaier** [comment]
Increased the margin-bottom from 2px to 4px for a bit easier reading:

image

**Review by bramkragten — APPROVED** [review]

**Pull request merged**